lib/DAC/DAC.cpp

```diff
@@ -4,137 +4,165 @@
 
 #include "../../include/definitions.h"
 
+#include <Helper.h>
+#include <I2CBus.h>
 #include <freertos/FreeRTOS.h>
 #include <freertos/task.h>
-#include <I2CBus.h>
-#include <Helper.h>
-#include "DAC.h"
 
+#include <Arduino.h>
+#include <SPI.h>
 #include <Wire.h> // I2C
-#include <stdio.h>
 #include <inttypes.h>
+#include <stdio.h>
 
-#define BASE_ADDR_CMD 0xA8
+#include "DAC.h"
+#include "DriverDisplay.h"
 
+#define BASE_ADDR_CMD 0xA8
 
 void dac_demo_task(void *pvParameter) {
 
-    // polling loop
-    uint8_t val = 0;
-    while(1) {
+  // // polling loop
+  // uint8_t val = 0;
+  // while (1) {
 
-        // set potentiometer value
-        printf("set potentiometer value=%d", val);
-        set_pot(val, POT_CHAN0);
-        set_pot(0xFF - val, POT_CHAN1); // negate
+  //   // set potentiometer value
+  //   printf("set potentiometer value=%d", val);
+  //   set_pot(val, POT_CHAN0);
+  //   set_pot(0xFF - val, POT_CHAN1); // negate
 
-        // sleep for 100ms
-        vTaskDelay(100 / portTICK_PERIOD_MS);
+  //   // sleep for 100ms
+  //   vTaskDelay(100 / portTICK_PERIOD_MS);
 
-        // increment timer
-        val += 10;
-    }
+  //   // increment timer
+  //   val += 10;
+  // }
```

Comment:
I wouldn't comment out the body of the functions. Either we do not need them anymore (->remove them completely) or we still keep them in (-> deactivate/comment out the calling location).

Reply:
clean up done